home/jobs/OMERO-build/config.xml

```diff
@@ -13,7 +13,7 @@
     </userRemoteConfigs>
     <branches>
       <hudson.plugins.git.BranchSpec>
-        <name>SPACEBRANCH/merge/trigger</name>
+        <name>SPACEBRANCH/merge/daily</name>
```

Comment:
The goal here was to run `OMERO-build` with an existing integration branch i.e. `develop/merge/daily` branch while we were working on setting up the SSH keys. I don't think this is meant to be committed though.